### PR TITLE
Submit Proposal with ledger

### PR DESCRIPTION
**Pull request by ztsai**

### CODE REVIEW

**Comment by ztsai** [comment]
#121

**Comment by ztsai** [comment]
also fixes #103

File: imports/ui/ledger/ledger.js [hunk 1]
```diff
@@ -396,9 +383,71 @@ export class Ledger {
...
+        txContext,
+        title,
+        description,
+        deposit
+    ) {
+        const txMsg = {
+            type: 'cosmos-sdk/MsgSubmitProposal',
+            value: {
+                content: {
+                    type: "cosmos-sdk/TextProposal",
```
Comment (kwunyeung) [inline imports/ui/ledger/ledger.js]: There are 3 types of proposal `text`, `parameter_change` and `software_upgrade`.

**Comment by kwunyeung** [comment]
tested.

**Pull request merged**